cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
+      free to give it a small name and add one in the same format as these.
+  secure_by_default: 
+    applies: ~
+    note: ~
+  security_by_obscurity: 
+    applies: ~
+    note: ~
+  serial_killer: 
+    applies: ~
+    note: ~
+  yagni: 
+    applies: ~
+    note: ~
+mistakes: 
+  answer: |
+      I think it's a coding mistake. Looking at the commit that fixed this problem, the issue
+      stemmed from lack of sanitization. Specifically, a URL containing @ allowed a colon to
+      be interpreted as part of the input for login. The committed fix addressed this directly.
   question: |
-    Did the feature that this vulnerability affected use inter-process
-    communication? IPC includes OS signals, pipes, stdin/stdout, message
-    passing, and clipboard. Writing to files that another program in this
-    software system reads is another form of IPC.
-
-    Answer should be boolean.
-  answer:
-  note:
-lessons:
+      In your opinion, after all of this research, what mistakes were made that
+      led to this vulnerability? Coding mistakes? Design mistakes?
+      Maintainability? Requirements? Miscommunications?
+      
+      Look at the CWE entry for this vulnerability and examine the mitigations
+      they have written there. Are they doing those? Does the fix look proper?
+      
+      Use those questions to inspire your answer. Don't feel obligated to answer
+      every one. Write a thoughtful entry here that those ing the software
+      engineering industry would find interesting.
+nickname: ~
+nickname_instructions: |
+    A catchy name for this vulnerability that would draw attention it. If the
+    report mentions a nickname, use that. Must be under 30 characters.
+    Optional.
+published_date: 2012-10-18
+published_instructions: |
+    Is there a published fix or patch date for this vulnerability?
+    Please enter your date in YYYY-MM-DD format.
+repo: ~
+reported_date: 2012-10-10
+reported_instructions: |
+    What date was the vulnerability reported to the security team? Look at the
+    security bulletins and bug reports. It is not necessarily the same day that the
+    CVE was created.  Leave blank if no date is given.
+    Please enter your date in YYYY-MM-DD format.
+reviews: []
+specification: 
+  answer: false
+  answer_note: |
+   I didn't find any relevant specifications.
+  instructions: |
+      Is there mention of a violation of a specification? For example,
+      an RFC specification, a protocol specification, or a requirements
+      specification.
+      
+      Be sure to check all artifacts for this: bug report, security
+      advisory, commit message, etc.
+      
+      The answer field should be boolean. In answer_note, please explain
+      why you come to that conclusion.
+subsystem: 
+  answer: "found in the directory of the fix commit: \"django/contrib/auth/\". a few files were modified to address password reset poisoning. b45c377f8f488955e0c7069cad3f3dd21910b071\n"
```

Comment:
The commit hash is already present in its own separate field, so it's not necessary here.

Reply:
Which files were modified?